tools/devctr/Dockerfile

```diff
@@ -72,6 +72,7 @@ RUN apt-get update \
         musl-tools \
         # needed for integration tests
         net-tools iproute2 iperf3 socat fdisk \
+        numactl \
```

Comment:
Have you pushed a new container image?

Reply:
No, I don't need this immediately, but if we are doing one soon it would be great to have this included.

Reply:
awesome, ok. I 'll pick it up with my PR then.